### DISCUSSION TITLE
Realtime not working with new local supabase instance

### DISCUSSION THREAD

**Discussion by jeegankones** [supabase/supabase#13519] [Questions]
I started prototyping my app just using the standard steps to set up a project at https://app.supabase.com and everything has been working great. RLS and Realtime work smoothly, had no problems connecting to my localhost or to my prod instance of my Vue app. 

However I wanted to start using a local supabase instance so that I could test features during development without messing up my prod data on my remote instance. I followed the documentation, installed the CLI, initialized everything, and ran `supabase db remote commit` to pull down all the schema changes from my remote.

This all worked fine, except when looking at the admin UI for my local instance I noticed that none of my tables had realtime enabled. So I manually went in and enabled realtime for all tables in my admin UI and saved those changes. But that didn't work. My local Vue app (connected to my local supabase instance using the API URL and anon key) still does not receive any realtime changes. I considered maybe I needed to stop and restart my local supabase instance for the changes to take effect, but stopping and restarting resets all of my tables to realtime disabled. 

What am I doing wrong here? Is there a trick to getting Realtime to work locally that I am missing?

**Comment by GaryAustin1** [comment]
Probably due to a new CLI version.
https://github.com/supabase/supabase/issues/13523
based on that report (not confirmed yet).

**Reply by jeegankones** [reply]
This does sound very similar to the problem I'm having. However he mentioned downgrading to 1.40.1 as a fix but so far I still have not gotten realtime to work on 1.40.1 either.
